ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -177,11 +177,28 @@ class ProfileMainScreen extends React.PureComponent<Props> {
   };
 
   private onPagoPAEnvironmentToggle = (enabled: boolean) => {
-    this.props.setPagoPATestEnabled(enabled);
-    this.props.showModal(
-      <AlertModal
-        message={I18n.t("profile.main.pagoPaEnvironment.alertMessage")}
-      />
+    Alert.alert(
```

Comment:
We should show the warning only when user tries to enable this feature.
As it is it always shows the warning, also when the user disables it.

Reply:
Sorry, I didn't think about it. Fixed :)